.github/workflows/labextension.yml

```diff
@@ -12,12 +12,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.11']
+        python-version: ['3.9', '3.13']
```

Comment:
Dask projects only support Python 3.10 and above. Is there any benefit to still supporting 3.9 here?

https://github.com/dask/dask/blob/87a177d866521a573f7a147cbee4dec7235f88f4/pyproject.toml#L29

Reply:
Not really!! I just used upstream JupyterLab as the reference as its lowest supported Python version is `3.9`. But I think if dask does not support it, I agree we should bump it to `3.10` directly.